ContentModeratorSDK/Contract/Review/ReviewVideoFramesResponse.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CognitiveServices.ContentModerator.Contract.Review
+{
+
+    public class ReviewVideoFramesResponse
+    {
+        public string ReviewId { get; set; }
+        public List<VideoFrame> VideoFrames { get; set; }
+    }
```

Comment:
Why do we need the Review Id property ? We get frames from only one Review at a time